app/templates/events/view/tickets/index.hbs

```diff
@@ -95,11 +95,11 @@
       <tr>
         {{#link-to 'events.view.tickets.orders.index' tagName='td'
         class='ui green inverted segment link'}}
-          <span>{{t 'Completed'}}</span>
```

Comment:
@geekyd @harshitagupta30 But that's what the main part is. @geekyd What do you think? Rest all looks good to me.